.werft/workspace-run-integration-tests.yaml

```diff
@@ -77,59 +30,140 @@ pod:
         secretKeyRef:
           name: slack-webhook-urls
           key: workspace_jobs
+    - name: USERNAME
+      valueFrom:
+        secretKeyRef:
+          name: integration-test-user
+          key: username
+    - name: USER_TOKEN
+      valueFrom:
+        secretKeyRef:
+          name: integration-test-user
+          key: token
+    volumeMounts:
+    - name: gcp-sa
+      mountPath: /mnt/secrets/gcp-sa
+      readOnly: true
+    - name: config
+      mountPath: /config
+      readOnly: false
     command:
-    - /bin/bash
+    - bash
     - -c
     - |
       set -euo
 
-      printf '{{ toJson .Annotations }}' > context.json
+      BRANCH="inte-test/"$(date +%Y%m%d%H%M%S)
+      export WERFT_CREDENTIAL_HELPER=/workspace/dev/preview/werft-credential-helper.sh
 
-      echo "[prep] receiving config..."
-      export GOOGLE_APPLICATION_CREDENTIALS="/config/gcloud/legacy_credentials/cd-gitpod-deployer@gitpod-core-dev.iam.gserviceaccount.com/adc.json"
-      echo "[prep] received config."
+      function cleanup ()
+      {
+        werft log phase "clean up" "clean up"
+        git push origin :$BRANCH | werft log slice "clean up"
```

Comment:
nit: can you double quote all variable names to prevent word splitting

Reply:
addressed